packages/cxl-ui/scss/global/cxl-section.scss

```diff
@@ -19,10 +21,6 @@ cxl-section#view-hero, cxl-section[theme~="cxl-hero"] {
   }
 
   &.wide-background-light-blue {
-    #view-hero .cxl-hero-titles {
```

Comment:
Unused code meant originally to position the instructor name caption on course pages

Reply:
@lkraav it's unrelated to the issue, but it's in the same file and could affect some pages